synr/ast.py

```diff
@@ -316,6 +316,7 @@ class TypeCall(Type):
 
     func_name: Union[Type, BuiltinOp]
     params: List[Type]
+    keyword_params: Dict[Type, Type]
```

Comment:
I think this should be `keyword_params: Dict[Id, Type]` as only identifiers are allowed as keywords. (I know regular `Call` is `Dict[Expr, Expr]`, but I think it should be `Dict[Id, Expr]`.

Reply:
done